pom.xml

```diff
@@ -141,28 +141,34 @@
           <mainClass>org.mitre.bonnie.cqlTranslationServer.Main</mainClass>
         </configuration>
       </plugin>
-      <!-- This builds the "uberjar" that can be used with java -jar to run -->
+       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy-dependencies</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/libs</outputDirectory>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-assembly-plugin</artifactId>
+        <artifactId>maven-jar-plugin</artifactId>
```

Comment:
Not a big deal, but my `maven` (3.8.4) is logging a warning about not explicitly specifying a version for the plugins used in `pom.xml`:

![image](https://user-images.githubusercontent.com/16297930/229532302-73ebedbc-d4c0-4e67-bc1b-5353321ada59.png)

Do we want to set this to just be the latest version for now to get rid of this warning?

Reply:
Oh dang. How did I miss that warning? Yeah, I'll add explicit versions.

Reply:
@mgramigna - e7f2601 sets plugin versions to their latest and also have a few small bonus config changes for said plugins.